### PR TITLE
USWDS-Site: Add changelog entries for #5358

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Added changelog entries for https://github.com/uswds/uswds/pull/5358

> Updated the ellipsis color to meet color contrast requirements.
> Updated styles to respect the value added to $theme-pagination-background-color.

## Preview link
[Pagination changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-5358/components/pagination/#changelog)

### CODE REVIEW

**Review by mejiaj — COMMENTED** [review]
Thoughts on combining these notes into one? They both come from a single code change. Otherwise LGTM.

**Comment by amyleadem** [comment]
@mejiaj That's a good question. I could go either way on combining them. I lean towards keeping them separate for a couple of reasons:
1. Only one item is a breaking change and I like being able to focus the breaking entry on related user action
2. The changes feel distinct to me and it feels easier to scan if they are separate entries

**Review by mahoneycm — APPROVED** [review]
LGTM! I'm indecisive about splitting into two or combining into one changelog. Maybe we should establish a pattern for if each changelog item is per _change_ or per _PR_.

**Comment by amyleadem** [comment]
@mahoneycm @mejiaj 
Our [changelog template README](https://github.com/uswds/uswds-site/blob/main/_data/changelogs/_CHANGELOG-README.md?plain=1#L32) currently recommends breaking up features into multiple data items: 
> If a single release item accomplished multiple tasks, break the tasks up into separate data items. The goal is to itemize and highlight user benefits and actions required.

I believe this was born out of some of the earlier discussions about changelogs. Happy to discuss this more if you are inclined to change the standard.

**Review by mejiaj — APPROVED** [review]
@amyleadem thanks for the feedback. Approving since there doesn't be a strong reason to combine them.

**Review by thisisdano — APPROVED** [review]
I support Amy's reasoning for a per-change approach to changelogs. A single PR may have multiple changelog-worthy items.

File: _data/changelogs/component-pagination.yml [hunk 1]
```diff
@@ -2,6 +2,22 @@ title: Pagination
...
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Updated styles to respect the value of `$theme-pagination-background-color`.
+  summaryAdditional: Users should confirm that pagination colors display as expected.
+  isBreaking: true
+  affectsStyles: true
+  githubPr: 5358
+  githubRepo: uswds
+  versionUswds: 3.6.0
```
Comment (thisisdano) [inline _data/changelogs/component-pagination.yml:12]:
```suggestion
  versionUswds: N.N.N
```

File: _data/changelogs/component-pagination.yml [hunk 2]
```diff
@@ -2,6 +2,22 @@ title: Pagination
...
+- date: NNNN-NN-NN
+  summary: Updated the ellipsis color to meet color contrast requirements.
+  summaryAdditional:
+  affectsAccessibility: true
+  affectsStyles: true
+  githubPr: 5358
+  githubRepo: uswds
+  versionUswds: 3.6.0
```
Comment (thisisdano) [inline _data/changelogs/component-pagination.yml:20]:
```suggestion
  versionUswds: N.N.N
```

**Review by thisisdano — COMMENTED** [review]
Updating the changelogs to add dummy version number

File: _data/changelogs/component-pagination.yml [hunk 3]
```diff
@@ -2,6 +2,22 @@ title: Pagination
...
+  githubRepo: uswds
+  versionUswds: N.N.N
+- date: NNNN-NN-NN
+  summary: Updated the ellipsis color to meet color contrast requirements.
+  summaryAdditional:
+  affectsAccessibility: true
+  affectsStyles: true
+  githubPr: 5358
+  githubRepo: uswds
+  versionUswds: 3.6.0
```
Comment (thisisdano) [inline _data/changelogs/component-pagination.yml:20]:
```suggestion
  versionUswds: N.N.N
```

**Pull request merged**